projects/services/public-api-service-typescript/build.sh

```diff
@@ -7,4 +7,14 @@ yarn install
 
 yarn run build
 
+echo "Copy mock server from client-sdk-javascript to run contract test..."
+cp ../../../client/client-sdk-javascript/src/orbs-api-interface.ts src
+cp ../../../client/client-sdk-javascript/test/mock-server.ts test
```

Comment:
This and the .gitignore above are too hacky.. can you reuse the code, without duplicating it like this?

Reply:
If we extract it as another library and then link it to both client and public api, then yes. If you think it’s necessary, I can do it in no time.